src/Collections/Sequences/LittleEndianNat.dfy

```diff
@@ -181,7 +181,7 @@ abstract module LittleEndianNat {
 
   /* The nat representation of a sequence can be calculated using the nat
   representation of its prefix. */
-  lemma LemmaSeqPrefix(xs: seq<uint>, i: nat)
+  lemma {:vcs_split_on_every_assert} LemmaSeqPrefix(xs: seq<uint>, i: nat)
```

Comment:
Is it OK to add `:vcs_split_on_every_assert` in the library?

Reply:
It's not ideal, IMO, but it's better than the status quo. I think it'd be better to make this proof easier in some other way, but it's not immediately clear how to do that. Any suggestions would be welcome!